requirements.txt

```diff
@@ -2,14 +2,13 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt requirements.in
+#    pip-compile --output-file=requirements.txt requirements.in
 #
-boto3==1.9.134
-botocore==1.12.134        # via boto3, s3transfer
+boto3==1.9.164
+botocore==1.12.164        # via boto3, s3transfer
 certifi==2019.3.9         # via requests
 chardet==3.0.4            # via requests
-docker-pycreds==0.4.0     # via docker
-docker==3.7.2
+docker==4.0.1
```

Comment:
Maybe skip upgrading docker if new version is not even following own docs?

Reply:
Moving to a separate PR